deps/v8/src/objects/map.cc

```diff
@@ -2126,7 +2120,7 @@ Handle<Map> Map::TransitionToDataProperty(Isolate* isolate, Handle<Map> map,
                                           StoreOrigin store_origin) {
   RuntimeCallTimerScope stats_scope(
       isolate, *map,
-      map->is_prototype_map()
+      map->IsDetached(isolate)
```

Comment:
This line had a conflict (the context above it has changed) - I only changed the affected line.